### DISCUSSION TITLE
The meaning and range of txpower

### DISCUSSION THREAD

**Discussion by jooray** [markqvist/Reticulum#201] [Help & Questions] [answered]
I've been checking the whole repository but was not able to find what is the range of possible txpower setting for a LoRA TTGO 1.6 device. Is there a range that is acceptable for HW?

**Comment by jooray** [comment, marked as answer]
Trial and error showed the maximum is 17. Does it make sense to add it to documentation or is it too device/country specific?

**Comment by markqvist** [comment]
Ah, that's a miss ;) It should definitely have been more clear. But you are right, it is 17dBm for that device. I'll get it added somewhere more obvious! Thanks for noting it!

**Reply by markqvist** [reply]
The information has been added in relevant places :)
